diff --git a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
index 013cd70a33a10b34df691a0ba7740d3404ebcc58..7b538dbf72f86f388ae66132c1907b49d65f7531 100644
--- a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
+++ b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
@@ -332,7 +332,7 @@ std::unique_ptr<VideoOverlayWindowViews> VideoOverlayWindowViews::Create(
   // constructor. It's important that the constructor be private, because it
   // doesn't initialize the object fully.
   auto overlay_window =
-      base::WrapUnique(new VideoOverlayWindowViews(controller));
+      base::WrapUnique(new BraveVideoOverlayWindowViews(controller));
 
   // The 2024 updated controls use dark mode colors.
   if (Use2024UI()) {
@@ -805,6 +805,7 @@ void VideoOverlayWindowViews::UpdateMaxSize(const gfx::Rect& work_area) {
 
   max_size_ = new_max_size;
 
+  BRAVE_UPDATE_MAX_SIZE
   if (!native_widget())
     return;
 
