diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index a0ff729c8efa676a41aae2694ef05e3172adca5c..d2dc9ff4b7bd73a66b609323f45f2c47090d3eeb 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1489,6 +1489,7 @@ source_set("extensions") {
       ]
     }
   }  # if (enable_extensions)
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions) {
