diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 217a86df8151cfa274c56f4082e7ed3ced0ac872..65535a04faabb2cbf8fe2a05745c903b48bf445d 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3565,6 +3565,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
