diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 662ac65225565a345f6607fe3687461014d44ec5..2fc95925b19b400eaa097f9d0751e7af777cd701 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -743,6 +743,7 @@ source_set("browser_sources") {
   if (is_chromeos_lacros) {
     deps += [ "//chromeos/startup" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
