diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index d8243ca31b40940f79892cf74a0d9dbbec7d8272..e6ba02fd69596b02fddc5111c73b66312aa28fb7 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -68,6 +68,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
