diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index 7dcdf8cef542bc31b9632ec6ea6cb662695bb112..e4efbf0dff709aaea8fed5cf217383055e566af7 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -212,6 +212,7 @@ tflite_static_library("tflite_builtin_op_resolver") {
     "//services/webnn/*",
     "//third_party/mediapipe/*",
   ]
+  import("//brave/third_party/tflite/tflite.gni") visibility += brave_tflite_builtin_op_resolver_visibility
 
   conditional_deps = [ ":tflite" ]
 }
