diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index 7440fb710f97477a51f0326b0b7c5a26fe8fb187..841d6e8277711503110bd0651601faaee4347931 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -91,6 +91,7 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/most_visited:build_ts",
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
+    "//chrome/browser/resources/settings:build_ts", "//brave/ui/webui/resources:build_ts"
   ]
   composite = true
   in_files = [
@@ -174,6 +175,7 @@ ts_library("build_ts") {
             "${node_modules}/@lit/reactive-element/reactive-element.d.ts",
             target_gen_dir),
   ]
+  import("//brave/third_party/lit/v3_0/sources.gni") visibility += brave_lit_build_ts_visibility
 }
 
 bundle_js("build_bundle") {
