diff --git a/BUILD.gn b/BUILD.gn
index 02249bd0d32a3534f64d1ef8489d5a05294f14a8..38bc2141dc9d455259a7692fb37f56fc404bade5 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -781,6 +781,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3276,6 +3277,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
