diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 4f249d9bc3bdc26d3cbb7a2b9ee03ffd6e55befa..73099bb414aef9b67f6bd28d03f9b9b7d0a290c2 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3342,6 +3342,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
